### PR TITLE
added simple method to compare counts of predicates between two graphs

### PR DESCRIPTION
**Pull request by justaddcoffee**
needed this for two things at the moment, so thought it might be generally useful

### CODE REVIEW

**Comment by kshefchek** [comment]
+1

File: tests/test_graph_utils.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import unittest
+import logging
+import rdflib
+from dipper.utils import GraphUtils
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
```
Comment (TomConlin) [inline tests/test_graph_utils.py]: other ingest all use `LOG = `  because it is short and a GLOBAL

File: tests/test_graph_utils.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        self.graph_object = rdflib.Graph()
+        self.graph_object2 = rdflib.Graph()        
+        self.graph_util = GraphUtils.GraphUtils
+        self.test_graph = self.graph_object.parse("tests/resources/graphutils/gu_test_graph.ttl", format="ttl")
+        self.test_graph_id = self.test_graph.identifier
+        self.test_graph2 = self.graph_object2.parse("tests/resources/graphutils/gu_test_graph2.ttl", format="ttl")
+        self.test_graph2_id = self.test_graph2.identifier
+        self.enemyOf = rdflib.term.URIRef('http://www.perceive.net/schemas/relationship/enemyOf')
+        self.name = rdflib.term.URIRef('http://xmlns.com/foaf/0.1/name')
+        return
```
Comment (TomConlin) [inline tests/test_graph_utils.py]: this is a point of no `return`

File: tests/test_graph_utils.py (hunk 1 above)
Comment (justaddcoffee) [reply tests/test_graph_utils.py]: fixed

File: tests/test_graph_utils.py (hunk 2 above)
Comment (justaddcoffee) [reply tests/test_graph_utils.py]: fixed

**Comment by TomConlin** [comment]
dipper/utils/GraphUtils.py 

    standard imports come before required or local imports
    trailing whitespace
    lines <= 88 cols (leftmost continuation preferred)
    one blank line between functions (two before class)
    omit empty return

**Comment by justaddcoffee** [comment]
okay @TomConlin think I've linted those issues

**Pull request merged**